frontend/webadmin/modules/uicommonweb/src/main/java/org/ovirt/engine/ui/uicommonweb/models/vms/UnitVmModel.java

```diff
@@ -2614,7 +2618,8 @@ public void initDisplayModels(Set<DisplayType> displayTypes, DisplayType selecte
         if (displayTypes.contains(selectedDisplayType)) {
             getDisplayType().setItems(displayTypes, selectedDisplayType);
         } else if (displayTypes.size() > 0) {
-            getDisplayType().setItems(displayTypes, displayTypes.iterator().next());
+            selectedDisplayType = behavior.getDefaultDisplayType(displayTypes);
```

Comment:
I do not like assigning values to functions's arguments, what about writing it like this?
```
getDisplayType().setItems(displayTypes, behavior.getDefaultDisplayType(displayTypes));
```